templates/kp_extraction/kp_extraction_emb.html

```diff
@@ -46,44 +55,78 @@
                     </div>
                 </div>
             </div>
-            <div class="col-md-1"></div>
-      </div>
+        </div>
     </div>
-
-
-    <div class='row' id="kpvizu_div">
+    <!-- Content area -->
+    <br>
+    <br>
+    <div class="container">
+        <div class="content-area">
+            <div class="content-stage container">
+                <div id="doc_block" class="col-xs-6">
+                    <div class="chat-area">
+                        <div id="html_doc">
+                            <!--  -->
```

Comment:
What's that?

Reply:
That I know, that there will be the result uploaded